README.md

```diff
@@ -20,27 +28,42 @@ You may use the contents of this repo in parts or in whole according to the 0BSD
 > OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH
 > THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-> **This repository does not accept Pull Requests at the moment.**
-
-## Components
-
-You can find a Docker Compose setup here with the following services:
-
-* Daml Platform:
-  * All-in-one Canton node (domain topology manager, mediator, sequencer and participant)
-  * HTTP JSON API service
-  * PostgreSQL database (used by the Canton node and the HTTP JSON API service)
-* Monitoring:
-  * Prometheus `2.x`
-  * Grafana `9.x`
-  * Node Exporter
-  * Loki + Promtail `2.x`
-
-Prometheus and Loki are [preconfigured as datasources for Grafana](./grafana/datasources.yml). You can add other
-services/exporters in the [Docker compose file](./docker-compose.yml) and scrape them changing the
-[Prometheus configuration](./prometheus/prometheus.yml).
-
-Use the `Makefile` — run `make help` for available commands!
```

Comment:
Not sure why this whole section got removed, if you want to implement your own monitoring, it's important to know which software is used in the monitoring stack and they interact with each other

Reply:
My bad.  Somehow I lost it.  I put it back.  Thank you for pointing this out.

Reply:
Actually, I see it there so I don't know why it doesn't appear for you.  It was embellished with more information though.